src/transformers/onnx/features.py

```diff
@@ -413,6 +415,12 @@ class FeaturesManager:
             "token-classification",
             onnx_config_cls="models.roformer.RoFormerOnnxConfig",
         ),
+        "segformer": supported_features_mapping(
+            "default",
+            "image-classification",
+            "semantic-segmentation",
```

Comment:
FYI @ydshieh discovered that the `adaptive_avg_pool2d` is not currently supported in ONNX. Not sure if that affects SegFormer or not, but if yes, you'll want to mimic what we did for Data2Vec: https://github.com/huggingface/transformers/pull/18587/files#diff-f4bad33d844c5d91b09dd0af27395ade640e907920b357daf125dd19458bdd60R243

Reply:
PS: Not found any string `adaptive` in `modeling_segformer.py` file.